### PR TITLE
add colors

### PR DESCRIPTION
**Pull request by YuriyKrasnobokiy**

### CODE REVIEW

**Comment by stas-yushkov** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -14,75 +14,209 @@
...
 
-          <div class="strength2">
-            <label class="strength2-label" for="strength2">Міцність</label>
-            <input
-              type="range"
-              id="strength2-progress"
-              class="JS-strength-input-2"
-              max="100"
-              value="15"
+        <li class="third">
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:117]: it`s unusable class. Just  for wrapper

File: index.html [hunk 2]
```diff
@@ -14,75 +14,209 @@
...
+          <div class="volume">
+            <label class="volume-label" for="volume">Об'єм</label>
             <input
               type="range"
-              id="volume2-progress"
+              id="volume-progress"
               class="JS-volume-input-2"
-              max="100"
-              value="30"
+              max="150"
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:107]: I changed it, but I think that 500 ml is too much.

File: index.html [hunk 3]
```diff
@@ -14,75 +14,209 @@
...
+            <div class="strength-percent">
+              <span class="number">45</span>
+              <span class="sign">%</span>
+            </div>
+          </div>
 
-          <div class="volume2">
-            <label class="volume2-label" for="volume2">Об'єм</label>
+          <div class="volume">
+            <label class="volume-label" for="volume">Об'єм</label>
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:102]: changed

File: index.css [hunk 4]
```diff
@@ -1,28 +1,159 @@
...
+button:focus {
+  background: rgb(52,125,57);
+  cursor: pointer;
+}
+
+.thumb {
+  padding-top: 70px;
+  display: flex;
+  justify-content: center;
+}
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.css:159]: ok

File: index.css [hunk 5]
```diff
@@ -1,28 +1,159 @@
 body {
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.css:1]: add

File: index.html [hunk 6]
```diff
@@ -14,75 +14,209 @@
         замініть браузер на нормальний:)
       </p>
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:15]: deleted

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:117]: I changed it

File: index.html [hunk 7]
```diff
@@ -14,75 +14,209 @@
...
+          <p class="current-number">4</p>
+        
+      <div class="strength">
+        <label 
+        class="strength-label" 
+        for="strength">
+        Міцність
+        </label>
+
+        <!-- <progress id="strength-progress" max="100" value="15"></progress> -->
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:164]: +

[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:164]: ok

File: index.html [hunk 8]
```diff
@@ -14,75 +14,209 @@
...
-      />
-      <p>Міцність коктейлю:</p>
-      <p class="JS-result">... (тицяй велику кнопку нижче) ...</p>
-      <button class="button JS-result-button">РОЗРАХУВАТИ</button>
+      </div>
+
+      <div class="results">
+        <div class="thumb">
+          <img 
+          src="./images/алкоШот.png"
```
[the remark this answers is not in the document]
Comment (YuriyKrasnobokiy) [reply index.html:210]: ok

**Review by stas-yushkov — APPROVED** [review]

**Pull request merged**